### ISSUE TITLE
Questions about data

### ISSUE THREAD

**Issue by lxy51** [uncbiag/uniGradICON#1]
Hello, thank you for your great creation. I wonder if you could give me a copy of the data used in the experiment.

**Comment by marcniethammer** [comment]
Thanks for your interest in our work. The OAI, HCP, and L2R-abdomen training datasets as well as all the test data we use are publicly available. Some of them require registering (and if applicable agreeing to existing use restrictions) before downloading the data. So we unfortunately cannot share the data here. It would also be too large.

**Comment by Promise13** [comment]
同学，你好，请问你找到该实验相关的数据集了吗？

**Comment by Promise13** [comment]
同学，你好！请问你找到该实验相关的数据集了吗？

**Comment by tushaarkataria** [comment]
Can you tell which folder you used for OAI dataset ?? On the NIH website I see multiple uploads of different sizes ? which one did you use ?

**Comment by lintian-a** [comment]
Hi @tushaarkataria ,

We got the OAI dataset a long time ago, which might be different from how the data is accessed today. 

If you want to conduct a comparison or replicate our result on the same data, I can send you the case IDs of the test set.

**Comment by LukeBudny** [comment]
Hello, many thanks for releasing the MultiGradICON paper!

I'm looking to replicate your work, but am having various issues with the gathering the data. These include:

Which of the COPDgene datasets are used (01-10)
Some details on which OAI/ABCD images were used, both have many terabytes of imaging available (I understand it was accessed a long time ago, but any help would be appreciated).
Were HCP Healthy Adults or HCP Development used, both have similar patient counts.

**Comment by lintian-a** [comment]
Hi @LukeBudny ,

Thanks for your interest in our work!

For the COPDGene dataset, are you referring to the test dataset or the training dataset?

**Comment by LukeBudny** [comment]
Thanks for the prompt reply @lintian-a !
I'd be interested in both as my current data source is the following:
https://med.emory.edu/departments/radiation-oncology/research-laboratories/deformable-image-registration/downloads-and-reference-data/index.html
As an aside, I have some questions regarding your project that relate to my research direction, is there a way I could message you directly?

**Comment by lintian-a** [comment]
Hi @LukeBudny ,

My email address is lintian@cs.unc.edu. Feel free to email me if you have any questions. 

We use the DirLab COPDGene cases for testing. For training, we obtain images directly from the COPDGene archive.
